src/wp-content/themes/twentynineteen/inc/helper-functions.php

```diff
@@ -11,7 +11,13 @@
  * Determines if post thumbnail can be displayed.
  */
 function twentynineteen_can_show_post_thumbnail() {
-	return apply_filters( 'twentynineteen_can_show_post_thumbnail', ! post_password_required() && ! is_attachment() && has_post_thumbnail() );
+	$show_post_thumbnail = ! post_password_required() && ! is_attachment() && has_post_thumbnail();
+	/**
+	 * Filters whether to show post thumbnail.
+	 *
+	 * @param bool $show_post_thumbnail Whether to show post thumbnail.
```

Comment:
Missing `@since x.x.x`

Reply:
None of the functions in `inc/helper-functions.php` have a `@since` note, but if added for the filter, it would have the theme name and two digits.
```
	 * @since Twenty Nineteen 1.0
```